packages/braintree-integration/src/braintree-paypal/braintree-paypal-payment-strategy.ts

```diff
@@ -316,4 +338,14 @@ export default class BraintreePaypalPaymentStrategy implements PaymentStrategy {
 
         throw new PaymentMethodFailedError(error.message);
     }
+
+    private isProviderError(error: unknown): boolean {
+        if (isPaypalCommerceProviderError(error)) {
+            const paypalProviderError = error?.errors?.filter((e: any) => e.provider_error) || [];
+
+            return paypalProviderError[0].provider_error?.code === '2046';
```

Comment:
Maybe it would be better to create a separate constant for '2046' error? How do you think?

Reply:
Sorry for the late response. I think it is not necessary, we use this code only here and nowhere else